lametro/events.py

```diff
@@ -318,6 +318,9 @@ def scrape(self, window=None) :
                                    url=approved_minutes['MatterAttachmentHyperlink'],
                                    media_type="application/pdf",
                                    date=self.to_utc_timestamp(approved_minutes['MatterAttachmentLastModifiedUtc']).date())
+                    e.extras['approved_minutes'] = True
+                else:
+                    e.extras['approved_minutes'] = False
```

Comment:
`approved_minutes` will only be populated if we get into the third branch of this control flow:

https://github.com/opencivicdata/scrapers-us-municipal/blob/33a01c55963e8bc49b87ee96b4c44e9f215b1ce0/lametro/events.py#L305-L323

How about defaulting to False before we get into this conditional section? (You can leave setting to True exactly where it is!)

Reply:
Great point, thanks!